kpc/admin.py

```diff
@@ -1 +1,7 @@
-from django.contrib import admin # noqa
+from django.contrib import admin
+from .models import Licensee
+
+
+@admin.register(Licensee)
+class LicenseeAdmin(admin.ModelAdmin):
+    pass
```

Comment:
Not super loving this "pass" in here--can we specify some list display fields so the admin interface is readable?